SECURITY.md

```diff
@@ -0,0 +1,23 @@
+# Security policy
+
+## Supported versions
+
+By the nature of Charmed HPC being still in a pre-alpha state, we currently maintain a latest/edge channel on all of our
+charms and snaps, and a latest/candidate on some of them. Hence, only those two channels will receive security updates at
+this point in time.
+
+## Reporting a vulnerability
+
+Please provide a description of the issue, the steps you took to
+create the issue, affected versions, and, if known, mitigations for
+the issue.
+
+The preferred way to report a security issue is through GitHub Security Advisories. See
+[Privately reporting a security
+vulnerability](https://docs.github.com/en/code-security/security-advisories/guidance-on-reporting-and-writing/privately-reporting-a-security-vulnerability)
+for instructions on reporting using GitHub's security advisory feature.
+
+The [Ubuntu Security disclosure and embargo
```

Comment:
Let's also this section since this is a common thing security firms will do. They'll set a disclosure deadline before they'll just announce the vulnerability whether or not we've actually fixed it:

```suggestion
If you have a deadline for public disclosure, please let us know. Our vulnerability management team intends to respond within 3 working days of your report. This project aims to resolve all vulnerabilities within 90 days.

The [Ubuntu Security disclosure and embargo
```